src/Arcus.Templates.Tests.Integration/WebApi/WebApiProject.cs

```diff
@@ -196,15 +196,29 @@ public static WebApiProject CreateNew(TestConfig configuration, WebApiProjectOpt
             return project;
         }
 
+        /// <summary>
+        /// Verifies if the specified file is part of the project.
+        /// </summary>
+        /// <param name="filename">The complete path and filename of the file, relative to the root folder of the project (without leading (back)slash).</param>
+        /// <returns>True if the file is present, otherwise false.</returns>
+        public bool ContainsFile(string filename)
+        {
+            Guard.NotNullOrWhitespace(filename, nameof(filename), "The filename parameter cannot be null or empty");
+
+            Console.WriteLine("Checking for file " + Path.Combine(ProjectDirectory.FullName, filename));
```

Comment:
Can we use the `ITestOutputHelper` from xUnit here instead? Or was this temporary and just for debugging purposes.

Reply:
Ah yes, I need to remove that.  Thx for pointing out.

Reply:
I used Console.WriteLine since I didn't want to bother including ITestOutputHelper, and it was just temporary.  However, now I've seen that there's an `ITestOutputHelper` available, so I improved the logging a bit :)